lib/Backend/GlobOpt.cpp

```diff
@@ -8909,21 +8910,30 @@ GlobOpt::TypeSpecialization(
             {
                 return instr;
             }
+
+            // need to run typespec if cannot const fold InlineMathAbs, or dst will be dead
+            if (instr->m_opcode != Js::OpCode::InlineMathAbs)
+            {
+                runTypeSpec = false;
```

Comment:
Best thing to do right now would be to remove the useless code here, and then do the proper fix for -off:constfold in master.

---
In reply to: [99963378](https://github.com/Microsoft/ChakraCore/pull/2267#discussion_r99963378) [](ancestors = 99963378,99457671)